docs/Get-MarkdownMetadata.md

```diff
@@ -78,60 +75,55 @@ external help file platyPS-help.xml
 schema             2.0.0
 ```
 
-Retrives metadata from all markdown file at the directory path provided.
+This command gets metadata from each of the markdown files in the .\docs folder.
 ## PARAMETERS
 
 ### -Path
-Path to markdown file or folder.
-Markdown files typically use extension .md
+Specifies an array of paths of markdown files or folders.
 
 
 ```yaml
 Type: String[]
 Parameter Sets: FromPath
-Aliases: 
+Aliases:
```

Comment:
This extra-spaces should not be edited by hands.
I'm afraid in current implementation they would be restored on the next `Update-MarkdownHelp` call.

It probably make sense to avoid putting them in the first place